tests/roles/pcp_cleanup/tasks/main.yaml

```diff
@@ -32,6 +32,8 @@
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    cd {{ install_yamls_path }}
-    for i in {1..3}; do make crc_storage_cleanup crc_storage && break || sleep 5; done
+    oc delete pvc ovn-data
```

Comment:
these delete commands should include `--ignore-not-found=true` to avoid errors when running on a clean environment

Reply:
done